packages/kbn-es-query/src/filters/build_filters/exists_filter.ts

```diff
@@ -6,26 +6,42 @@
  * Side Public License, v 1.
  */
 
+import { get } from 'lodash';
 import type { IndexPatternFieldBase, IndexPatternBase } from '../../es_query';
-import type { Filter, FilterMeta } from './types';
-
-export type ExistsFilterMeta = FilterMeta;
-
-export interface FilterExistsProperty {
-  field: any;
-}
+import type { FieldFilter, Filter, FilterMeta } from './types';
 
+/** @public */
 export type ExistsFilter = Filter & {
-  meta: ExistsFilterMeta;
-  exists?: FilterExistsProperty;
+  meta: FilterMeta;
+  exists?: {
+    field: string;
+  };
 };
 
-export const isExistsFilter = (filter: any): filter is ExistsFilter => filter && filter.exists;
+/**
+ * @param filter
+ * @returns `true` if a filter is an `ExistsFilter`
+ *
+ * @public
+ */
+export const isExistsFilter = (filter: FieldFilter): filter is ExistsFilter =>
+  get(filter, 'exists');
```

Comment:
This function doesn't explicitly return a boolean. Should this be:
```suggestion
  get(filter, 'exists') != null;
```

If it's valid for `filter.exists` to be `0`, then this function seems it would return the wrong value. Perhaps this is not a big deal, and also the same issue existed in the previous code.

Reply:
Fixed it by replacing `get` with `has` which does exactly that